tests/test_sitl.py

```diff
@@ -39,6 +43,34 @@ def test_version_list():
     models.sort()
     assert_equals(expected, models)
 
+
 def test_download():
     from dronekit_sitl import download
-    download('copter','3.3', None)
\ No newline at end of file
+    download('copter','3.3', None)
+
+
```

Comment:
yeah I noticed we just didn't care before, we should start improving on style, hence my comments, thanks for fixing this